frontend/src/modules/search/components/SearchBox.js

```diff
@@ -65,7 +65,21 @@ export class SearchBoxBase extends React.Component<InternalProps, State> {
     constructor(props: InternalProps) {
         super(props);
 
+        this.state = {
+            search : '',
+            statuses: {},
+            extras: {},
+            tags: {},
+            timeRange: null,
+            authors: {},
+        };
+        
+        this.searchInput = React.createRef();
+    }
+
+    updateFiltersFromURLParams = () => {
         const statuses = this.getInitialStatuses();
+        const { props } = this
```

Comment:
Please add ; at the end of the line and we're good! :)

Reply:
AND please move this line to the top of the function and add a blank line after it.